src/MOCK_DB_DATA/library-items/tooltip-popover/popover.ts

```diff
@@ -0,0 +1,73 @@
+import { LibraryItemIds } from '../library-item-ids';
+import { LibraryItem, LibraryItemType } from 'src/app/views/shared/models/library-item.model';
+import { LibraryItemConstants as CONST } from '../library-item-constants';
+
+export const Popover: LibraryItem[] = [{
+  id: LibraryItemIds.Popover,
+  title: 'Popover',
+  type: LibraryItemType.ngBootstrap,
+  exampleHtml: getExampleHtml(),
+  codeHtml: getCodeHtml(),
+  documentationHtml: getDocumentationHtml()
+}];
+
+
+/*
+ * Return value is ignored if LibraryItemType is not html
+ */
+function getExampleHtml(): string {
+  return ``;
+}
+
+/*
+ * Return empty string to use a copy of exampleHtml as codeHtml.
+ * Return null to remove Code from library-item.
+ */
+function getCodeHtml(): string | null {
+  return `
+<button type="button"
+        class="fhi-tag fhi-tag--category"
+        [autoClose]="'outside'"
+        [ngbPopover]="popContent">Aktiver popover</button>
+
+<p class="mt-6">
+  Popover kan også brukes i løpende tekst.
+
+  <a role="button"
+     class="popover-trigger"
```

Comment:
Sorry for all the extra touble 😇

Reply:
...and we still have a issue with where to put ng-bootstrap styling. But we'll leave it as it is for now.